addons/website/views/snippets.xml

```diff
@@ -4,11 +4,7 @@
 <template id="s_title" name="Title">
     <section class="pt32 pb32">
         <div class="container">
-            <div class="row s_nb_column_fixed">
-                <div class="col-lg-12 s_title pt16 pb16" style="text-align: center;">
-                    <h1 class="s_title_default"><font style="font-size: 62px;">Your Site Title</font></h1>
-                </div>
-            </div>
+            <h1 class="s_title_default" style="font-size: 62px; text-align: center;">Your Site Title</h1>
```

Comment:
Does it work on all browser to add the font-size directly on the h1? I think that if we explicitely add a `<font/>` element is because the summernote editor does not handle this correctly otherwise...

Reply:
It's actually the opposite. I did this because that's how the editor does it, using a font tag makes it uneditable.

Reply:
Nope, broken with this on my side, please recheck. The editor adds a <font> stuff when editing the font-size. If a font already exists, it uses it. If not it creates it. With your code, you end up having this:
```xml
<h1 style="font-size: 62px"><font style="font-size: 13px;">TITLE</font></h1>
```
which breaks the layout since the line-height uses the 62px size.

I agree that the new editor will ideally work in both cases, so yours will be better of course. But right now, it seems broken with it. Check different browsers maybe?

Reply:
You're right. I remembered something breaking, but it was actually the text-align style, which has to be on the h1 to work.